torch/quantization/fx/quantize.py

```diff
@@ -620,6 +685,16 @@ def load_arg(a):
                 observed_graph, load_arg)
 
 
+        self.modules = dict(model.named_modules())
+
+        # TODO: refactor this to a separate function
+        matches = self._find_matches(
+            observed_graph, self.modules, self.patterns, standalone_module_names,
+            standalone_module_classes, custom_module_classes)
+        quants = self._find_quants(observed_graph, self.modules, matches)
+
+        observed_graph = handle_copy_nodes(observed_graph, matches, quants, self.activation_post_process_map, self.modules)
```

Comment:
one source of confusion is that L620:L685 are inlined, but this line is in a separate function.  Thoughts about either inlining both or having both in separate functions, just so it's consistent on where this style of "go through all the nodes and create a new graph" code is happening?

Reply:
sounds good

Reply:
will do this in a follow up PR